src/main/java/io/xstefank/wildlfy/bot/format/checks/DescriptionCheck.java

```diff
@@ -0,0 +1,45 @@
+package io.xstefank.wildlfy.bot.format.checks;
+
+import io.xstefank.wildlfy.bot.config.RegexDefinition;
+import org.kohsuke.github.GHPullRequest;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class DescriptionCheck implements Check{
```

Comment:
we need checkstyle I see :)

Reply:
Can u help to install checkstyle on my idea tomorrow?